### ISSUE TITLE
Semantic Hybrid search giving results of normal semantic serach

### ISSUE THREAD

**Issue by pgroo** [Azure/cognitive-search-vector-pr#75]
In comparing the c# sdk with the python sdk, we have noticed, with identical requests and parameters that the semantic hybrid search is returning the expected result on python but is returning the identical result to the ordinary semantic search on the c# call.

ex c# params:
var vector = new SearchQueryVector { KNearestNeighborsCount = 10, Fields = "embeddings", Value = queryEmbeddings.ToArray() };
                var searchOptions = new SearchOptions
                {
                    Vector = vector,
                    Size = 10,
                    QueryType = SearchQueryType.Semantic,
                    QueryLanguage = QueryLanguage.EnUs,
                    SemanticConfigurationName = SemanticSearchConfigName,
                    QueryCaption = QueryCaptionType.Extractive,
                    QueryAnswer = QueryAnswerType.Extractive,
                    QueryCaptionHighlightEnabled = true,
                    Select = { "{column_name}" },
                };
SearchResults<SearchDocument> response = await searchClient.SearchAsync<SearchDocument>(query, searchOptions);

**Comment by jelledruyts** [comment]
I've seen this too in v11.5.0-beta.3 of the .NET SDK: when the query text parameter is _not_ null, the vector will _not_ be sent to the search service and therefore it performs a regular text search rather than a hybrid search.

So changing the last line to this will keep the vector, but then you obviously don't have hybrid search:

SearchResults response = await searchClient.SearchAsync(null, searchOptions);

So in other words, there seems to be no way to perform a _hybrid_ search with the .NET SDK today.

**Comment by rahulm132002** [comment]
In the SearchQueryVector, setting the value of the KNearestNeighborsCount to Null enables you to do the semantic hybrid search, as follows:

 SearchQueryVector { KNearestNeighborsCount = null, Fields = "embeddings", Value = queryEmbeddings.ToArray()

**Comment by jelledruyts** [comment]
No, sorry, that doesn't seem to make a difference, the vector still isn't sent to the search service.
